pages/Settings.jsx

```diff
@@ -0,0 +1,89 @@
+import { useEffect, useState } from "react";
+import styles from "@/styles/pages/settings.module.scss";
+
+export default function Settings() {
+	const [theme, setTheme] = useState("default");
+	const [fontSize, setFontSize] = useState(-1);
+
+	useEffect(() => {
+		if (localStorage.getItem("theme")) {
+			if (theme === "default") {
```

Comment:
A flicker that happens when the page loads (both in theme and font size) because of the use of the default value - the component first renders on the default value and only updates the correct values on the 2nd/3rd rerenders (caused by the effect).
As far as I can see, there is no solution that solves this flicker that doesn't involves the server, so we'll leave the flicker for now.

Reply:
noted